.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "modules/PX4Firmware"]
 	path = modules/PX4Firmware
-	url = git://github.com/ArduPilot/PX4Firmware.git
+	url = https://github.com/crashmatt/Firmware.git
```

Comment:
Thanks Francisco.  Travis is giving an error I understand. Semaphore is stuck with the module change as you suggest.